tfjs-core/src/kernel_names.ts

```diff
@@ -18,21 +18,24 @@
 // tslint:disable: variable-name
 // Unfortunately just enabling PascalCase per file (tslint:enable:
 // allow-pascal-case) doesn't work.
-import {NamedTensorInfoMap} from './kernel_registry';
+import {NamedTensorInfoMap, TensorInfo} from './kernel_registry';
 import {PixelData} from './types';
 
+export const AddN = 'AddN';
+export type AddNInputs = TensorInfo[];
```

Comment:
We need to have an exception here, AddNInputs is an array of Tensors. We don't have a fixed number of Tensors, so we cannot use NamedTensorInfoMap. Cast this input as NamedTensorMap is fine, because array is basically an object with index as key.